lib/ajv/createAjv.js

```diff
@@ -1,12 +1,29 @@
-const Ajv = require('ajv');
+const Ajv = require('ajv').default;
 
+const addFormats = require('ajv-formats');
 const addByteArrayKeyword = require('./keywords/byteArray/addByteArrayKeyword');
 
 /**
  * @return {ajv.Ajv}
  */
 function createAjv() {
-  const ajv = new Ajv();
+  const ajv = new Ajv({
+    strictTypes: true,
+    strictTuples: true,
+    strictRequired: true,
+    addUsedSchema: false,
+    strict: true,
+  });
+
+  ajv.addVocabulary([
+    'ownerId',
+    'documents',
+    'protocolVersion',
+    'indices',
+    '$commit',
```

Comment:
It just misstype https://github.com/dashevo/js-dpp/blob/e173e8bfb745da17f1c014c6d805d988a0c44a0a/schema/identity/publicKey.json#L24